doc/advanced/override-default-entrypoint.rst

```diff
@@ -0,0 +1,41 @@
+Overriding the default entry point
+----------------------------------
+
+In some cases, it may be necessary to override the default main entry point of
+an Ocaml program. For example, this is the case if you want to let your program
+handle argument wildcards expansion on Windows.
+
+Fortunately, the ``foreign_stubs`` stanza can be leveraged for that usage.
```

Comment:
Adding a pointer to the `foreign_stubs` reference might help the reader to dig further if they want to.

Reply:
Good idea, done.